### DISCUSSION TITLE
Arbitrary dynamic value in % doesn't work!

### DISCUSSION THREAD

**Discussion by sibasishm** [tailwindlabs/tailwindcss#12951] [Help] [answered]
**What version of Tailwind CSS are you using?**

^3.3.5

**What build tool (or framework if it abstracts the build tool) are you using?**
Next.js 13.4.5

**What version of Node.js are you using?**

For example: v16.16.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Reproduction URL**

Tailwind Play link: https://play.tailwindcss.com/k8WoiSDkuB?layout=horizontal

**Describe your issue**

The arbitrary value for left position in % works in tailwind play (v3.4.1) but doesn't work in browser! What am I missing!
My markup looks like:
```
<div
  className={`w-0 h-0 border-l-[8px] border-l-transparent border-b-[15px] border-b-[#474b9e] border-r-[8px] border-r-transparent -mt-2 absolute left-[${score}%]`}
/>
										```

**Comment by MichaelAllenWarner** [comment]
Read this: https://tailwindcss.com/docs/content-configuration#dynamic-class-names

**Comment by sibasishm** [comment]
> Read this: https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Ofcourse! The dynamic value. Thanks @MichaelAllenWarner 
I have shifted the same to style tag and made it work but is there anyway to do it using Tailwind CSS?

**Comment by MichaelAllenWarner** [comment, marked as answer]
You can do `style="--score: {score}%"` (using whatever syntax is appropriate for your templating language), and then use the utility class `left-[--score]` (which in Tailwind is shorthand for `left-[var(--score)]`).

**Comment by MichaelAllenWarner** [comment]
I guess you're using React, so maybe: `` style={{ '--score': `${score}%` }} ``
